src/draw/grouping.typ

```diff
@@ -337,17 +337,24 @@
   )
 },)
 
-/// Creates a new anchor for the current group. This element can only be used inside a group otherwise it will panic. The new anchor will be accessible from inside the group by using just the anchor's name as a coordinate.
+/// Creates a new anchor.
```

Comment:
We should keep some description of how to use it and what happens in and out of a group